postgres/db_setup_draft.sql

```diff
@@ -218,7 +218,6 @@ CREATE TABLE courses(
     owner INTEGER REFERENCES users(uid),
     course_invite VARCHAR(32),
     created BIGINT DEFAULT now_as_millis(),
-    UNIQUE(invite),
     UNIQUE(course_id)
```

Comment:
@colinmegill what about this missing attribute? postgres printed an error for this